src/assets/Enums.ts

```diff
@@ -320,6 +329,14 @@ export const LETTER_TYPE: string[] = [
     "Trailer rejection"
 ];
 
+export const PARAGRAPH_IDS: string[] = [
+    "3",
+    "4",
+    "5",
+    "6",
+    "7"
+];
+
```

Comment:
These are numbers in the VTM front-end (and the doc-gen services) - should they be numbers/integers here too?